### PR TITLE
Alex.changes

### PR DESCRIPTION
**Pull request by alkorchagin**

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+file 1 haha
```
Comment (arkolesen) [inline file1.txt:1]: WHAT?

File: file2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+2nd file lol
```
Comment (arkolesen) [inline file2.txt:1]: @alkorchagin LOL? what is it?

File: file1.txt (hunk 1 above)
Comment (alkorchagin) [reply file1.txt:1]: go fuck

Comment (arkolesen) [reply file1.txt:1]: Block this merge. @alkorchagin  Stupid developer

**Pull request merged**